openedx_webhooks/tasks/github.py

```diff
@@ -413,17 +416,27 @@ def github_community_pr_comment(pull_request, jira_issue, people=None):
     people = {user.lower(): values for user, values in people.items()}
     pr_author = pull_request["user"]["login"].decode('utf-8').lower()
     created_at = parse_date(pull_request["created_at"]).replace(tzinfo=None)
+    missing_committers = []
+    if commits:
+        for committer in commits:
+            try:
+                if commits[committer]["author"]["login"] not in people:
+                    missing_committers.append(commits[committer]["author"]["login"])
+            except TypeError:
+                pass
     # does the user have a valid, signed contributor agreement?
     has_signed_agreement = (
         pr_author in people and
-        people[pr_author].get("expires_on", date.max) > created_at.date()
+        people[pr_author].get("expires_on", date.max) > created_at.date() and not
+        missing_committers
```

Comment:
Looks like we consider the expires_on for the pr_author, but not for the commiters?  I smell a function in our future...